FilterModule.adoc

```diff
@@ -44,3 +46,44 @@ harmonic profile of your sounds and make it evolve over time.
 More on that later in this chapter.
 
 image::imgs/20180413-144131.png[]
+
+===== Moog Filter
+
+image::imgs/filter-moog.png[]
+
+The Moog filter type is an attempt to emulate the renowned diode ladder filter found in classic Moog synthesizers. 
```

Comment:
Technically the moog ladder filter was the transistor based ladder. Since they patented that design, but everyone else wanted the same sound people ended up with diode ladder filters for analog synths developed by other groups. 

It's a fun bit of trivia :)